forum/qa-theme/SnowFlat/qa-theme.php

```diff
@@ -247,38 +247,30 @@ public function body_content()
         $this->widgets('full', 'top');
         $this->header();
 
-        $address = qa_path_absolute(qa_request());
+        $request = qa_request();
+        $divideRequest = explode('/', $request);

```

Comment:
ale to tylko takie moje zboczenie: http://php.net/manual/en/function.parse-url.php